src/integration-test/resources/application-test.properties

```diff
@@ -22,3 +22,9 @@ spring.main.banner_mode=off
 
 spring.datasource.username=root
 spring.datasource.password=
+
+# Using the Scheduler as Authentication Service
+pa.scheduler.rest.url=http://localhost:8080/rest
```

Comment:
Can you double check with @paraita how he solved the url catalog on the studio and see if you can apply the same fix here?